### PR TITLE
Fix reference error on search click

### PR DESCRIPTION
**Pull request by liorbentov**
Steps to reproduce:

Run server.
Search for tutorial.
Let the page load.
Check browser console.

### CODE REVIEW

**Comment by Animesh-Ghosh** [comment]
Provide at least a description or refer this issue from a related issue.

File: app/static/app/js/custom.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
-setTimeout(function focusMe() {
+function focusMe() { window.setTimeout( function() {
 	document.getElementById("focus-here").scrollIntoView();
```
Comment (Animesh-Ghosh) [inline app/static/app/js/custom.js]: Throws TypeError: document.getElementById(...) is null.

Use #46 inside **`focusMe`**'s definition and it should be good to go.

**Comment by liorbentov** [comment]
I think #46 should be merged first, am I right?

**Comment by Animesh-Ghosh** [comment]
Closing issue as this had been fixed.

**Pull request closed without merging**